lib/src/screens/wallet_connect/services/walletkit_service.dart

```diff
@@ -562,9 +578,11 @@ abstract class WalletKitServiceBase with Store {
     final filteredPairings = allPairings.where(
       (pairing) {
         bool isInCurrentTopics = currentTopicsForWallet.contains(pairing.topic);
-        bool isActive = pairing.active;
+        // bool isActive = pairing.active;
+        // bool hasSession = sessions.any((session) => session.pairingTopic == pairing.topic);
 
-        return isInCurrentTopics && isActive;
+        // return isInCurrentTopics && isActive;
+        return isInCurrentTopics;
```

Comment:
why are those no longer needed?

Reply:
Mainly because we no longer need them

For the isActive check, the pairing can be valid but inactive, and we want to be able to either disconnect properly or reconnect. The previous check filters out those types of pairing completely so we don't have the option to take action on them.

Same for the hasSession check, we have another way to retrieve sessions when needed, and a pairing should not depend on that.